<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="6" passed="4" failed="2" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-02-01T12:40:07 IST" name="Suite" finished-at="2024-02-01T12:40:11 IST" duration-ms="3620">
    <groups>
    </groups>
    <test started-at="2024-02-01T12:40:07 IST" name="Test" finished-at="2024-02-01T12:40:11 IST" duration-ms="3620">
      <class name="com.demo.tests.PutTests">
        <test-method signature="updateEmployeeDetails()[pri:0, instance:com.demo.tests.PutTests@65c7a252]" started-at="2024-02-01T12:40:10 IST" name="updateEmployeeDetails" finished-at="2024-02-01T12:40:10 IST" duration-ms="40" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- updateEmployeeDetails -->
      </class> <!-- com.demo.tests.PutTests -->
      <class name="com.demo.tests.DeleteTests">
        <test-method signature="deleteEmployeeDetails()[pri:0, instance:com.demo.tests.DeleteTests@4d154ccd]" started-at="2024-02-01T12:40:10 IST" name="deleteEmployeeDetails" finished-at="2024-02-01T12:40:10 IST" duration-ms="61" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[
expected: 200
 but was: 404]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: 
expected: 200
 but was: 404
at com.demo.tests.DeleteTests.deleteEmployeeDetails(DeleteTests.java:30)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- deleteEmployeeDetails -->
      </class> <!-- com.demo.tests.DeleteTests -->
      <class name="com.demo.tests.GetTests">
        <test-method signature="getEmployees()[pri:0, instance:com.demo.tests.GetTests@18df8434]" started-at="2024-02-01T12:40:10 IST" name="getEmployees" description="To get employee data" finished-at="2024-02-01T12:40:10 IST" duration-ms="74" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getEmployees -->
        <test-method signature="getEmployeeDetails()[pri:0, instance:com.demo.tests.GetTests@18df8434]" started-at="2024-02-01T12:40:10 IST" name="getEmployeeDetails" finished-at="2024-02-01T12:40:11 IST" duration-ms="972" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[
expected: "Kuhlman"
 but was: "Thiel"]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: 
expected: "Kuhlman"
 but was: "Thiel"
at com.demo.tests.GetTests.getEmployeeDetails(GetTests.java:38)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getEmployeeDetails -->
      </class> <!-- com.demo.tests.GetTests -->
      <class name="com.demo.tests.PostTests">
        <test-method signature="postEmployeeData()[pri:0, instance:com.demo.tests.PostTests@60bd273d]" started-at="2024-02-01T12:40:08 IST" name="postEmployeeData" finished-at="2024-02-01T12:40:10 IST" duration-ms="2544" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- postEmployeeData -->
        <test-method signature="postEmployeeDataFromExternalFile()[pri:0, instance:com.demo.tests.PostTests@60bd273d]" started-at="2024-02-01T12:40:08 IST" name="postEmployeeDataFromExternalFile" finished-at="2024-02-01T12:40:10 IST" duration-ms="2544" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- postEmployeeDataFromExternalFile -->
      </class> <!-- com.demo.tests.PostTests -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
